phone-test/src/pages/CallDetail/CallDetail.tsx

```diff
@@ -59,11 +101,29 @@ export const CallDetail: React.FC<CallDetailProps> = ({ call, onClick }) => {
             </Typography>
             <Typography variant="caption">{date}</Typography>
           </Box>
+          <Spacer space="s">
```

Comment:
This is a simplistic implementation. It works, but I'm not really happy with the look of it.